app/controllers/miq_policy_controller.rb

```diff
@@ -942,6 +942,8 @@ def folder_get_info(folder_node)
   # Build the audit object when a profile is saved
   def build_saved_audit(record, add = false)
     name = record.respond_to?(:name) ? record.name : record.description
+    name = record.description if record.kind_of?(MiqPolicy)
```

Comment:
The conditionals for setting `name` seem a bit clumsy. It's not a big deal, but there's probably a cleaner way to do it.

Reply:
Ok, will change it as:
```
    name = if record.kind_of?(MiqPolicy)
             record.description
           else
             record.respond_to?(:name) ? record.name : record.description
           end
```